src/experimental/information_estimates.cpp

```diff
@@ -389,7 +384,8 @@ kt_double InformationEstimates::calculateLaserMutualInformation(kt_double const
      * Return:
         * kt_double: Laser mutual information
     */
-    return map_info - curr_info;
+    // return map_info - curr_info;
+    return m_info_grid.sum();
```

Comment:
@kmilo7204 nit: given it is this simple, does it make sense to have a function for it?